### PR TITLE
CAS-1217 VirtualEvent Overview screen

### PR DESCRIPTION
**Pull request by samiuelson**
https://stream-io.atlassian.net/browse/CAS-1217

Overview screen of the virtual event app.

https://user-images.githubusercontent.com/4527432/132042621-5ec9a7d9-fb24-42d8-9694-3e35e2442b7a.mp4

### CODE REVIEW

File: virtualevent-demo/src/main/res/layout/activity_home.xml [hunk 1]
```diff
@@ -3,33 +3,31 @@
...
-        app:layout_constraintRight_toRightOf="parent"
-        app:itemIconTint="@color/tab_item_foreground"
-        app:itemTextColor="@color/tab_item_foreground"
-        app:menu="@menu/bottom_nav_menu" />
+    android:layout_height="match_parent">
 
     <fragment
         android:id="@+id/nav_host_fragment_activity_home"
         android:name="androidx.navigation.fragment.NavHostFragment"
         android:layout_width="match_parent"
```
Comment (bychkovdmitry) [inline virtualevent-demo/src/main/res/layout/activity_home.xml:11]: I would make this 0dp as well

File: virtualevent-demo/src/main/res/layout/fragment_overview.xml [hunk 2]
```diff
@@ -1,22 +1,423 @@
...
-    xmlns:tools="http://schemas.android.com/tools"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".feature.overview.OverviewFragment">
+    android:layout_height="match_parent">
 
-    <TextView
-        android:id="@+id/text_home"
+    <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (bychkovdmitry) [inline virtualevent-demo/src/main/res/layout/fragment_overview.xml:6]: Duplicate import

File: virtualevent-demo/src/main/res/layout/fragment_overview.xml [hunk 3]
```diff
@@ -1,22 +1,423 @@
...
+
+        </GridLayout>
+
+        <TextView
+            android:id="@+id/schedule"
+            style="@style/OverviewHeader"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="18dp"
+            android:text="Schedule"
```
Comment (bychkovdmitry) [inline virtualevent-demo/src/main/res/layout/fragment_overview.xml:254]: Even in a sample app, I would still extract text strings

**Review by bychkovdmitry — APPROVED** [review]

File: virtualevent-demo/src/main/res/layout/activity_home.xml (hunk 1 above)
Comment (samiuelson) [reply virtualevent-demo/src/main/res/layout/activity_home.xml]: done

File: virtualevent-demo/src/main/res/layout/fragment_overview.xml (hunk 3 above)
Comment (samiuelson) [reply virtualevent-demo/src/main/res/layout/fragment_overview.xml]: done

**Pull request merged**